Functions/Quantification/xASL_quant_HadamardDecoding.m

```diff
@@ -46,8 +46,8 @@
 
 % Specify the TimeEncodedMatrix
 if isfield(xQ,'TimeEncodedMatrix') && ~isempty(xQ.TimeEncodedMatrix)
-	if (size(xQ.TimeEncodedMatrix,1)+1) ~= size(xQ.TimeEncodedMatrix,2)
-		error('TimeEncodedMatrix must be square (N-1)xN ');
+	if (size(xQ.TimeEncodedMatrix,1)) ~= size(xQ.TimeEncodedMatrix,2)
```

Comment:
You can remove the first double brackets

Reply:
Will do.

Reply:
Done in 63feb017